scripts/migration/migrate_registered_meta.py

```diff
@@ -0,0 +1,77 @@
+""" Changes existing registered_meta on a node to new schema layout
+required for the prereg-prize
+"""
+import re
+import ast
+import sys
+import logging
+
+from nose.tools import *
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def get_registered_nodes():
+    return models.Node.find(
+        Q('is_registration', 'eq', True)
+    )
+
+
+def main(dry_run=True):
+    init_app(routes=False)
+    count = 0
+
+    if not dry_run:
+        scripts_utils.add_file_logger(logger, __file__)
+        logger.info("Iterating over all registrations")
+        models.MetaSchema.remove()
+
+    for node in get_registered_nodes():
+        for schema in node.registered_meta:
```

Comment:
Instead of

```python
for key in my_dict:
   value = my_dict.get(key)
```

You can write:

```python
for key, value in my_dict.items():
    # ...
```